mail_template_multi_attachment/models/mail_template.py

```diff
@@ -30,7 +30,13 @@ def generate_email(self, res_ids, fields):
         if isinstance(res_ids, int):
             multi_mode = False
             results = {res_ids: results}
-        self.generate_attachments(results)
+        active_model = self.env.context.get("active_model")
+        model_id = self.env.context.get("active_id")
+        if active_model and model_id:
+            partner_lang = self.env[active_model].browse(model_id).partner_id.lang
```

Comment:
You don't know what is going to be the `model_id`, not if it will have a `partner_id` on it. it will lead to failure

Reply:
Hello @leemannd , I was discussing it with @PaoloYam . A new version of this PR is comming to avoid this ;-)

Reply:
Didn't knew. Thank you